### DISCUSSION TITLE
Wiki

### DISCUSSION THREAD

**Discussion by Duke-M-commits** [ReactiveDrop/reactivedrop_translations#71] [Ideas]
I created a Wiki on my fork.
https://github.com/DukeDukeEm/reactivedrop_translations/wiki
Then I realized this part of the git project can't be included in pull requests.

How about you create a wiki on your side which users can edit?

**Comment by BenLubar** [comment]
I've enabled the wiki, not sure if there are any other steps required.

**Comment by Duke-M-commits** [comment]
The readme is getting pretty long.
How about we move parts of it into the git-wiki?
Also, i read, you could integrate it into the repo itself.
https://stackoverflow.com/questions/6941688/how-to-integrate-a-github-wiki-into-the-main-project#6941730

**Reply by Blueberryy** [reply]
mediawiki can be good to migrate too

**Reply by Duke-M-commits** [reply]
Nope, it‘s github flavored markdown with no blob-storage, it‘s fine (doesn’t blow up the repo and easy to read without a parser).

**Reply by Duke-M-commits** [reply]
Don‘t get me wrong wikitext is awesome and i love it, but i think every usage has its fitting tool and there is more than one awesome tool, and in this case IMHO markdown is the way to go (and github seems to think so too).
